README.md

```diff
@@ -243,6 +243,11 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // Optional unless running as a protected resource.
                 "client_secret": "xxxxxxxxxxxxxxx",
 
+                // The secret used when validating a JWT signed with a symmetric algorithm (e.g. HS256).
+                // Used if provided, otherwise, defaults to validating with "client_secret".
+                // If provided, it MUST be base64url encoded.
```

Comment:
The line length causes a horizontal scrollbar to appear. Please adjust.

I noticed the `user_mapping` stanza causes that too. Please adjust that as well.